src/plugins/ni/testmod_ni.c

```diff
@@ -449,12 +449,16 @@ int main (int argc, char ** argv)
 
 	TEST (ver);
 	TEST (new);
+#ifndef __SANITIZE_ADDRESS__
```

Comment:
Yes, it is an issue in the nickel library (nickel.c:413 if I remember correctly). There is even a valgrind fix for this issue in the nickel.c (around line 380 I think). It would be possible to move the "fix" to the nickel.c, but I'm not sure if that is a good idea (as it's not maintained by us)?